lib/virt_autotest/utils.pm

```diff
@@ -116,6 +132,42 @@ sub is_kvm_host {
     return check_var("SYSTEM_ROLE", "kvm") || check_var("HOST_HYPERVISOR", "kvm") || check_var("REGRESSION", "qemu-hypervisor");
 }
 
+#retrun 1 if libvirt 9.0- is running which monolithic libvirtd is the default service
+sub is_monolithic_libvirtd {
+    unless (is_alp) {
+        return 1 if script_run('rpm -q libvirt-libs | grep -e "libs-9\.0" -e "libs-[1-8]\."') == 0;
+    }
+}
```

Comment:
It seems that this subroutine does not return explicitly for all cases. Better to return 0 or 1 for all cases ?

Reply:
yes, thanks.